blosc/schunk.c

```diff
@@ -377,14 +430,17 @@ int blosc2_schunk_free(blosc2_schunk *schunk) {
 }
 
 
-/* Create a super-chunk out of a serialized frame (no copy is made). */
-blosc2_schunk* blosc2_schunk_open_sframe(uint8_t *sframe, int64_t len) {
-  blosc2_frame* frame = blosc2_frame_from_sframe(sframe, len, false);
+/* Create a super-chunk out of a frame buffer */
+blosc2_schunk* blosc2_schunk_from_buffer(uint8_t *framebuf, int64_t len, bool copy) {
+  blosc2_frame_s* frame = frame_from_framebuf(framebuf, len, false);
```

Comment:
Maybe rename to `frame_from_buffer`?

Reply:
I think I like using `cframe` for contiguous frames.  I did the change already.